script.js

```diff
@@ -306,3 +394,90 @@ function handleStatesLayer(geojson) {
 
   return statesLayer;
 }
+
+function handleCountiesCartoLayer(geojson) {
+  // style layer based on whether their moratorium has passed
+  const layerOptions = {
+    style: feature => {
+      const yesOptions = {
+        color: "#4dac26",
+        fillColor: "#b8e186",
+        fillOpacity: fillOpacity,
+        weight: strokeWeight
+      };
+
+      const noOptions = {
+        color: "#d01c8b",
+        fillColor: "#f1b6da",
+        fillOpacity: fillOpacity,
+        weight: strokeWeight
+      };
+
+      const splitOptions = {
```

Comment:
I was thinking that if there was a duplicate entry where the response was yes and no, then the styling would be gray.  But this will be removed per the new data cleaning method you mentioned.